.readthedocs.yml

```diff
@@ -11,7 +11,8 @@ search:
   ranking:
     # Deprecated content
     api/v1.html: -1
+    api/v2.html: -1
```

Comment:
I think we should keep this around for now, since we aren't fully on v3

Reply:
yeah, good point